File: fastapi_server/routes/home_route.py
from fastapi import FastAPI

def mount(app: FastAPI):
    """
    Mount routes to the FastAPI application.
    This function is called by the server bootstrap process.
    """
    @app.get("/")
    async def home():
        return {"message": "Hello from autoloaded route!", "framework": "fastapi"}

File: fastify_server/routes/home_route.mjs
/**
 * Mount routes to the Fastify application.
 * This function is called by the server bootstrap process.
 * @param {import('fastify').FastifyInstance} server
 */
export async function mount(server) {
  server.get("/", async (request, reply) => {
    return { message: "Hello from autoloaded route!", framework: "fastify" };
  });
}
